Document usage of CC_HEALTH_CHECK_PATH in blue-green deployment. (#294)
CleverCloud/documentation: layouts/shortcodes/content/url_healthcheck.md

The deployment process checks all paths. All of them must reply with a `200 OK` response code.

> When no envrionment variable is defined the default is to get the root path, `/`.

Reviewer: ```suggestion
By default, when no environment variable is defined, the root path `/` will be checked.
```

> By default, when no environment variable is defined, the root path `/` will be checked.

Reviewer 2: Specify what checks the root path and which path is that: 
```suggestion
By default, when no environment variable is defined, the monitoring check your repository root path `/`.
```
